tests/integration/security/reachability_test.go

```diff
@@ -106,6 +106,10 @@ func TestReachability(t *testing.T) {
 						if src == rctx.Naked {
 							return opts.Target == rctx.Naked
 						}
+						// Headless service with sidecar injected, autMtls does not apply.
+						if opts.Target == rctx.Headless {
```

Comment:
i am only expecing this line change, why other test config file are also changed?